README.md

```diff
@@ -52,26 +52,26 @@ gulp.task('templates', function() {
 
 All options supported by the [Jade API](http://jade-lang.com/api/) are supported
```

Comment:
So is everything good for the pull request?

On Fri, May 6, 2016, 7:37 PM Jamen Marzonie <notifications@github.com>
wrote:

> In README.md
> <https://github.com/jamen/gulp-jade/pull/117#discussion_r62404551>:
>
> > @@ -52,26 +52,26 @@ gulp.task('templates', function() {
> >
> >  All options supported by the [Jade API](http://jade-lang.com/api/) are supported
>
> @thisboyiscrazy <https://github.com/thisboyiscrazy> That should be fine
> for now. I'll go ahead and change it in the future if I don't notice
> (because it should redirect too).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jamen/gulp-jade/pull/117/files/92d1ac0d347b63e560f0dd092c09d12cccdc1951#r62404551>
>